frontend/layouts/project-page/contact/component.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import Button from 'components/button';
+import LayoutContainer from 'components/layout-container';
+import { ContactProps } from 'layouts/project-page/contact/types';
+
+export const Contact: React.FC<ContactProps> = ({ project }: ContactProps) => {
+  const { contact_email: email, contact_phone: phone } = project?.project_developer;
+  const [contactInfoModalOpen, setIsContactInfoModalOpen] = useState<boolean>(false);
+  // TODO: add contactInfoModal component when set contactInfoModalOpen to true
+  return (
+    (!!phone || !!email) && (
```

Comment:
One caveat here is that the modal has to list not only the associated `project_developer` contact details, but also the contact details of the `involved_project_developers`. 

There's a quick and dirty implementation to get an array of contact details to be displayed in the modal [here](https://github.com/Vizzuality/heco-invest/blob/develop/frontend/containers/project-details/favorite-contact/component.tsx#L46) (which probably should be extracted into a helper function now, since it's reused in the project header). 

We have a modal to display all the contact details [here](https://github.com/Vizzuality/heco-invest/tree/develop/frontend/containers/social-contact/contact-information-modal). 

We don't need to do it now, we can extract this into a separate task (and see if the contacts/modal is of use in other components as well).